src/js/node/events.js

```diff
@@ -1,8 +1,11 @@
 // Reimplementation of https://nodejs.org/api/events.html
+
 // Reference: https://github.com/nodejs/node/blob/main/lib/events.js
 const { throwNotImplemented } = require("$shared");
+const { FixedQueue } = require("$fixed_queue");
```

Comment:
Can we lazy load this?
```suggestion
let lazyFixedQueue;
```
later:
```ts
const FixedQueue = (lazyFixedQueue ??= require("internal/fixed_queue"))
```
(also, the $shared notation is very old and i was too lazy to migrate all uses to the new resolver. if that import doesnt build please nag me, i dont want to add more `$` prefixed imports)